app/pages/view-solution/template.njk

```diff
@@ -1,60 +1,20 @@
-
 {% extends 'includes/layout.njk' %}
-{% from 'components/button/macro.njk' import button %}
 {% from 'components/back-link/macro.njk' import backLink %}
-{% from 'sections/view-solution-header/macro.njk' import viewSolutionHeader %}
-{% from 'sections/view-solution-description/macro.njk' import viewSolutionDescription %}
-{% from 'sections/view-solution-contact-details/macro.njk' import viewSolutionContactDetails %}
-{% from 'sections/view-solution-capabilities/macro.njk' import viewSolutionCapabilities %}
-{% from 'components/view-section-heading/macro.njk' import viewSectionHeading %}
-{% from 'components/tag/macro.njk' import tag %}
+
+{% from 'sections/view-solution/macro.njk' import viewSolution %}
 
 {% block body %}
-  <div data-test-id="view-solution-page" class="nhsuk-width-container nhsuk-u-margin-top-4 nhsuk-u-margin-bottom-9">
-    <div data-test-id="view-solution-page-back-link" class="nhsuk-u-margin-bottom-5">
+  <div class="nhsuk-width-container nhsuk-u-margin-top-5">
+      <div data-test-id="view-solution-page-back-link" class="nhsuk-u-margin-bottom-5">
```

Comment:
🙄

Reply:
lol

Reply:
this one is right

Reply:
really? 
Should it not be indented just 2 spaces?

Reply:
I knew I should have written that nunjucks linter 😄